contracts/Schema.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.6.0 <0.9.0;
+/**
+ * @title Schema
+ * @dev Schema contract
+ */
+contract SchemaRegistry {
+
+    struct Schema {
+        address issuer;
+        string name;
+        bytes32 hash;
+        bytes body;
+        uint256 timestamp;
+    }
+
+    mapping(string => bytes32) nameHash;
+    mapping(bytes32 => Schema) hashSchema;
+
+    function getHashFromBytes(bytes memory schemaBody) private pure returns (bytes32) {
+        return keccak256(schemaBody);
+    }
+
+    /**
+     * @dev save is function to store schema
+     * @param schemaName - name of the schema
+     * @param schemaBody - json schema representation in bytes
+     */
+    function save(string memory schemaName, bytes memory schemaBody) public {
```

Comment:
How about `store` name for the function, anyway updating is not possible (maybe it should be?)

Reply:
Save is ok. Update is not possible.

Reply:
save should be ok, as it is actually save operation